packages/medusa/src/loaders/admin.ts

```diff
@@ -3,25 +3,23 @@ import { Express } from "express"
 
 type Options = {
   app: Express
-  configModule: ConfigModule
+  adminConfig: ConfigModule["admin"]
 }
 
 type IntializedOptions = Required<
   Pick<AdminOptions, "path" | "disable" | "outDir">
 > &
   AdminOptions
 
-export default async function adminLoader({ app, configModule }: Options) {
-  const { admin } = configModule
-
+export default async function adminLoader({ app, adminConfig }: Options) {
   const adminOptions: IntializedOptions = {
     disable: false,
     path: "/app",
     outDir: "./build",
-    ...admin,
+    ...adminConfig,
```

Comment:
since this is optional 
```suggestion
    ...(adminConfig || {}),
```

Reply:
Will it matter? Because, you can spread `undefined` values too.

Reply:
TIL 😄